docs/sphinx/applications/python/adapt-vqe.ipynb

```diff
@@ -0,0 +1,449 @@
+            "mycc.kernel()\n", "\n", "print('R-CCSD energy = ', mycc.e_tot)\n",
+            "                \n", "# Compute the electron integrals\n", "\n",
+            "h1e_ao = mol.intor(\"int1e_kin\") + mol.intor(\"int1e_nuc\")\n",
+            "h1e=reduce(np.dot, (mf.mo_coeff.T, h1e_ao, mf.mo_coeff))\n",
+            "h2e_ao = mol.intor(\"int2e_sph\", aosym='1')\n",
+            "h2e=ao2mo.incore.full(h2e_ao, mf.mo_coeff)\n",
+            "h2e=h2e.transpose(0,2,3,1)\n",
+            "nuclear_repulsion = mf.energy_nuc()\n"
+        ]
+    }, {
+        "cell_type":
+            "markdown",
+        "metadata": {},
+        "source": [
+            "## Jordan Wigner: \n", "\n",
+            "Convert fermionic Hamiltonian to qubit Hamiltonian."
+        ]
+    }, {
+        "cell_type":
+            "code",
+        "execution_count":
+            11,
+        "metadata": {},
+        "outputs": [{
+            "name":
+                "stdout",
+            "output_type":
+                "stream",
+            "text": [
+                "Total number of pauli hamiltonian terms:  19\n",
+                "[0.1659278503377034+0j] IZZI\n",
+                "[0.12062523483390415+0j] IZIZ\n",
+                "[0.17141282644776912+0j] IZII\n",
+                "[0.17441287612261572+0j] IIZZ\n",
+                "[0.04530261550379926+0j] YXXY\n",
+                "[0.04530261550379926+0j] XYYX\n", "[0+0j] XYXY\n",
+                "[0.12062523483390415+0j] ZIZI\n",
+                "[-0.04530261550379926+0j] XXYY\n", "[0+0j] XXXX\n",
+                "[-0.22343153690813433+0j] IIIZ\n", "[0+0j] YXYX\n",
+                "[0.16868898170361205+0j] ZZII\n",
+                "[-0.09706626816763109+0j] IIII\n",
+                "[0.1659278503377034+0j] ZIIZ\n",
+                "[-0.04530261550379926+0j] YYXX\n",
+                "[0.17141282644776912+0j] ZIII\n", "[0+0j] YYYY\n",
+                "[-0.22343153690813433+0j] IIZI\n", "\n"
+            ]
+        }],
+        "source": [
+            "from adapt_vqe.hamiltonian import jordan_wigner_fermion, generate_molecular_spin_ham_restricted\n",
+            "\n", "# Get the spin molecular hamiltonian\n",
+            "obi, tbi, nuclear_repulsion = generate_molecular_spin_ham_restricted(h1e,h2e, nuclear_repulsion)\n",
+            "\n",
+            "# Convert the fermionic Hamiltonian to a qubit Hamiltonian\n",
+            "tolerance=1e-15\n",
+            "spin_ham = jordan_wigner_fermion(obi, tbi, nuclear_repulsion,tolerance)\n",
+            "print('Total number of pauli hamiltonian terms: ',spin_ham.get_term_count())\n",
+            "\n", "print(spin_ham)"
+        ]
+    }, {
+        "cell_type":
+            "markdown",
+        "metadata": {},
+        "source": [
+            "## UCCSD operator pool\n", "\n", "### Single excitation\n",
+            "$$ T_{ij} = \\frac{i}{2} (X_i Y_j - Y_i X_j) \\prod_{p=i+1}^{j-1} Z_p$$\n",
+            "\n", "### Double excitation\n",
```

Comment:
This is how it renders in the docs preview - doesn't seem right:
![image](https://github.com/user-attachments/assets/2ed0d918-cc00-4df6-bc97-854e64995c31)